force-app/main/default/lwc/scoreInformation/scoreInformation.html

```diff
@@ -0,0 +1,33 @@
+<template>
+    <lightning-layout>
+        <lightning-layout-item padding="around-small">
+            <div class="header-column">
+                <p class="field-title" title="Score">Score</p>
+                <p>Attempt: {attemptId} Score: {score}%</p>
```

Comment:
For practice look into using a variable for this and something called a template literal.